paymentsheet/src/test/.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.{kt,kts}]
+disabled_rules = import-ordering
+max_line_length=140
```

Comment:
We are using 100 everywhere, isn't 140 too long?

Reply:
Unfortunately we are not restricting to 100 characters in payments-core because there would be so many errors.  The editorconfiig there is not applied because it has an extra 'i' in the file name.